networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="roll-back-to-ovn-kubernetes"]
+= Roll back to the OVN-Kubernetes network plugin
```

Comment:
Mostly you've got "Rolling" except for here it's just "Roll". Suggest syncing this heading up on "Rolling" as well.